poetry.lock

```diff
@@ -345,16 +345,16 @@ graph = ["objgraph (>=1.7.2)"]
 profile = ["gprof2dot (>=2022.7.29)"]
 
 [[package]]
-name = "eq_translations"
-version = "4.9.2"
-description = "Translations infrastructure for EQ Questionnaire Runner"
+name = "eq-translations"
+version = "1.0.0"
```

Comment:
We had some discussions in the team about this - it's pulling across whatever is set in the eq-translations `pyproject.toml` file since we moved to poetry for package management. Will raise a ticket to investigate moving away from `setup.py` and realigning our repos from this

Reply:
Ticket raised 👍